src/test/java/hu/dungeonhunter/TestFight.java

```diff
@@ -36,5 +36,34 @@ public void championDefeatTest() {
 
         assertThat(fight.nextTurn()).as("There should be no next turn.").isFalse();
         assertThat(lowHpChampion.isDefeat()).as("Champion should have been defeated.").isTrue();
+    }
+
+    @Test
+    public void runningAwayAndTheChampionWillDieTest() {
+        Fight fight = new Fight();
+        Champion lowHpChampion = new Champion(1);
+        fight.setChampion(lowHpChampion);
+
+        fight.runningAway();
+
+        assertThat(fight.nextTurn()).as("There should be no next turn.").isFalse();
+        assertThat(lowHpChampion.isDefeat()).as("Champion should have been defeated.").isTrue();
+    }
+
+    @Test
+    public void runningAwaySuccessfulEscapeTest(){
+        Fight fight = new Fight(1);
+        Champion highHpChampion = new Champion(10);
+        fight.setChampion(highHpChampion);
+
+        fight.runningAway();
+
+        assertThat(fight.getMonsterCounter()).as("monsterCounter test to decrease in value").isEqualTo(0);
+        assertThat(fight.nextTurn()).as("There should be no next turn, because no more monster.").isFalse();
```

Comment:
Also add assertion to check that champion is not defeated.

Reply:
Thank you, it's done!